scripts/build.ts

```diff
@@ -7,14 +7,18 @@ import { mkdir, readFile, readdir, rm, writeFile } from 'fs/promises';
 import MagicString from 'magic-string';
 import path from 'path';
 import { rollup } from 'rollup';
+import { fileURLToPath } from 'url';
 import { webfont } from 'webfont';
 import { toArray } from '../src/shared/util';
 import { components } from './components';
 import { parallel, runTask, series } from './tasks';
 import type { ComponentProto } from '../src/types';
 import type { Plugin, SourceMapInput } from 'rollup';
 
-const SRC_DIR = path.join(__dirname, '../src/');
+const __filename = fileURLToPath(import.meta.url);
+const dirname = path.dirname(__filename);
```

Comment:
Just curious why you didn't stick to `__dirname`? It seems like there can be fewer changes. 🙂

Reply:
I got an error when trying to use `__dirname` - I imagine it's some kind of node version difference?

Reply:
`__dirname` is not available in ESM, you need to define it yourself.

Reply:
> I got an error when trying to use __dirname - I imagine it's some kind of node version difference?

Hm. I don't get any errors. That is really weird. I'm using Node v22.14.0